src/main/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/EvidenceService.java

```diff
@@ -499,11 +479,9 @@ private static String factorBasedSummaryLabel(final ExperimentDesign experimentD
                                                                    AssayGroup testAssayGroup) {
         return Stream.of("organism part", "cell line", "cell type").flatMap(
                 x -> {
-                    SampleCharacteristic s =
-                            experimentDesign.getSampleCharacteristic(testAssayGroup.getFirstAssayId(), x);
+                    var s = experimentDesign.getSampleCharacteristic(testAssayGroup.getFirstAssayId(), x);
```

Comment:
Instead of a letter 's' as a variable name. I think it would be nice if we add a word(relevant to context).
'x ->' it would be nice If add a word instead of x. x means, is it experment expression?

Reply:
None, it’s the values of the stream. To be fair, in this case I found it was sort of justified since the values `x` takes are right in the preceding line, but I changed it anyway for the sake of clarity.